.github/workflows/build/action.yml

```diff
@@ -9,9 +9,13 @@ runs:
     if: runner.os == 'Linux'
     uses: "./.github/workflows/build/linux"
 
-  - name: Build (OSX)
-    if: runner.os == 'MacOS'
-    uses: "./.github/workflows/build/osx"
+  - name: Build (OSX) Intel
+    if: runner.os == 'MacOS' && env.TARGET == 'x86_64'
+    uses: "./.github/workflows/build/osx-intel"
+
+  - name: Build (OSX) Silicon
+    if: runner.os == 'MacOS' && env.TARGET == 'arm64'
+    uses: "./.github/workflows/build/osx-silicon"
```

Comment:
I don't think its necessary to separate the workflows for Intel/Silicon - for the Dissolve refactoring (which is almost working!) I just used a single osx workflow, and didn't need to specify architectures because we're doing native compilation now so the defaults are acceptable. Your two main osx workflows are basically identical, and this "stub" action is version-agnostic in the sense of macos-latest vs macos-14, so you can just have one action to cope with everything.